cmd/opampsupervisor/README.md

```diff
@@ -9,15 +9,25 @@ The design is still undergoing changes, and as such this implementation may chan
 The supervisor is currently undergoing heavy development and is not ready for any serious use. However, if you would like to test it, you can follow the steps below:
 
 1. Download the [opamp-go](https://github.com/open-telemetry/opamp-go) repository, and run the OpAMP example server in the `internal/examples/server` directory.
+
+   ```shell
+   git clone git@github.com:open-telemetry/opamp-go.git
+   cd internal/examples/server
```

Comment:
If you just cloned it, you need to `cd` into the cloned directory too, right? :slightly_smiling_face: 
```suggestion
   cd opamp-go/internal/examples/server
```

Reply:
changed